manifest-generation/runtime-aci-tile.yml

```diff
@@ -0,0 +1,412 @@
+  - reference: ".properties.network_node_service_ip_pool_start"
+    label: IP pool start address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_ip_pool_end" 
+    label: IP pool end address
+    description: date_type:network_address
+  - reference: ".properties.network_node_service_subnets" 
+    label: Network node service subnets
+    description: date_type:string
+
+runtime_configs:
+- name: aci-containers-tile 
+  runtime_config: |
+    releases: 
+    - name: aci-containers
+      version: '0.0.9'
+    addons:
+    - name: network-setup-addon 
+      jobs:
+      - name: network-setup
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+    - name: openvswitch-switch-addon 
+      jobs:
+      - name: ovs
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+    - name: agent-ovs-addon 
+      jobs:
+      - name: agent-ovs
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+      properties:
+        apic:
+          vmm_policy: (( .properties.apic_vmm_policy.value ))
+          vmm_domain: (( .properties.apic_vmm_domain.value ))
+          vmm_controller: (( .properties.apic_vmm_controller.value ))
+          encap_type: (( .properties.apic_encap_type.value ))
+    - name: aci-cf-host-agent-addon
+      jobs:
+      - name: aci-cf-host-agent
+        release: aci-containers
+      include:
+        jobs:
+        - name: rep
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+      properties:
+        apic:
+          vmm_domain: (( .properties.apic_vmm_domain.value ))
+          vmm_controller: (( .properties.apic_vmm_controller.value ))
+          encap_type: (( .properties.apic_encap_type.value ))
+          policy_vrf_tenant: (( .properties.apic_policy_vrf_tenant.value ))
+          policy_vrf: (( .properties.apic_policy_vrf.value ))
+          service_encap: (( .properties.apic_service_encap.value ))
+        etcd:
+          ca_cert: (( $ops_manager.ca_certificate ))
+          client_cert: (( ..etcd-aci-tile.properties.generated_etcd_cert.cert_pem ))
+          client_key: (( ..etcd-aci-tile.properties.generated_etcd_cert.private_key_pem))
+        network:
+          subnet: (( .properties.network_subnet.value ))
+    - name: aci-cf-controller-addon
+      jobs:
+      - name: aci-cf-controller
+        release: aci-containers
+      include:
+        jobs:
+        - name: bbs
+          release: diego
+        stemcell:
+        - os: ubuntu-trusty
+      properties:
+        api:
+          port: (( .properties.networking_aci_api_port.value ))
+          path_prefix: "/networking-aci"
+        apic:
+          hosts: ["172.28.184.150"]
```

Comment:
There are a lot of hardcoded values in this "properties" section. Please replace them with parameters.